src/main/java/forceitembattle/ForceItemBattle.java

```diff
@@ -159,6 +164,19 @@ public void onEnable() {
             world.setGameRule(GameRule.KEEP_INVENTORY, getConfig().getBoolean("settings.keepinventory"));
             world.setGameRule(GameRule.ANNOUNCE_ADVANCEMENTS, false);
         });
+
+        if(this.getConfig().isConfigurationSection("descriptions")) {
+            ConfigurationSection configurationSection = this.getConfig().getConfigurationSection("descriptions");
+            Set<String> materialKeys = configurationSection.getKeys(false);
+
+            materialKeys.forEach(keys -> {
+                List<String> descriptions = configurationSection.getStringList(keys);
+                getItemDifficultiesManager().getDescriptionItems().add(new DescriptionItem(Material.valueOf(keys), descriptions));
```

Comment:
Is this null-save? What happens if there is no description for an item in the config (on accident)?

Reply:
This should be null-save. I tried to empty all lines of one item (only the -), no errors, its like an empty line. Otherwise if I just leave the item there without any '-' then the material is deleted when loading the file. Nevertheless I added a check for configurationsection 'description' != null. Just to be safe that there is even a "descriptions" key in yml